lib/ecl/fortio.c

```diff
@@ -488,6 +488,17 @@ void fortio_data_fseek(fortio_type* fortio, offset_type data_offset, size_t data
     }
 }
 
+int fortio_fclean(fortio_type * fortio) {
+  long current_pos = ftell(fortio->stream);
+  if(current_pos == -1)
+    return -1;
+
+  int flush_status = fflush(fortio->stream);
+  if(flush_status != 0)
```

Comment:
Yes, because the position in the stream after a flush is undefined. And in particular, it results in funky behaviour with clang on OS X..